### ISSUE TITLE
Java deprecation warings in version 5.0.0

### ISSUE THREAD

**Issue by under3415** [googleads/googleads-mobile-flutter#1036]
I've installed version `5.0.0`, and now there are even more java deprecation warnings!
This is a continuation of [issue 940](https://github.com/googleads/googleads-mobile-flutter/issues/940)

One warning is an existing one from previous issue for `getApplicationInfo`. Others are new related to deprecated `mediationNetworkExtrasProvider`. 

```

\AppData\Local\Pub\Cache\hosted\pub.dev\google_mobile_ads-5.0.0\android\src\main\java\io\flutter\plugins\googlemobileads\FlutterAdRequest.java:39: warning: [deprecation] MediationNetworkExtrasProvider in io.flutter.plugins.googlemobileads has been deprecated
  @Nullable private final MediationNetworkExtrasProvider mediationNetworkExtrasProvider;
                          ^
\AppData\Local\Pub\Cache\hosted\pub.dev\google_mobile_ads-5.0.0\android\src\main\java\io\flutter\plugins\googlemobileads\FlutterAdRequest.java:51: warning: [deprecation] MediationNetworkExtrasProvider in io.flutter.plugins.googlemobileads has been deprecated
    @Nullable private MediationNetworkExtrasProvider mediationNetworkExtrasProvider;
                      ^
\AppData\Local\Pub\Cache\hosted\pub.dev\google_mobile_ads-5.0.0\android\src\main\java\io\flutter\plugins\googlemobileads\FlutterAdRequest.java:101: warning: [deprecation] MediationNetworkExtrasProvider in io.flutter.plugins.googlemobileads has been deprecated
        @Nullable MediationNetworkExtrasProvider mediationNetworkExtrasProvider) {
                  ^
\AppData\Local\Pub\Cache\hosted\pub.dev\google_mobile_ads-5.0.0\android\src\main\java\io\flutter\plugins\googlemobileads\FlutterAdRequest.java:149: warning: [deprecation] MediationNetworkExtrasProvider in io.flutter.plugins.googlemobileads has been deprecated
    protected MediationNetworkExtrasProvider getMediationNetworkExtrasProvider() {
              ^
\AppData\Local\Pub\Cache\hosted\pub.dev\google_mobile_ads-5.0.0\android\src\main\java\io\flutter\plugins\googlemobileads\FlutterAdRequest.java:190: warning: [deprecation] MediationNetworkExtrasProvider in io.flutter.plugins.googlemobileads has been deprecated
      @Nullable MediationNetworkExtrasProvider mediationNetworkExtrasProvider,
                ^
\AppData\Local\Pub\Cache\hosted\pub.dev\google_mobile_ads-5.0.0\android\src\main\java\io\flutter\plugins\googlemobileads\FlutterAdManagerAdRequest.java:89: warning: [deprecation] MediationNetworkExtrasProvider in io.flutter.plugins.googlemobileads has been deprecated
      @Nullable MediationNetworkExtrasProvider mediationNetworkExtrasProvider,
                ^
\AppData\Local\Pub\Cache\hosted\pub.dev\google_mobile_ads-5.0.0\android\src\main\java\io\flutter\plugins\googlemobileads\AdMessageCodec.java:74: warning: [deprecation] MediationNetworkExtrasProvider in io.flutter.plugins.googlemobileads has been deprecated
  @Nullable private MediationNetworkExtrasProvider mediationNetworkExtrasProvider;
                    ^
\AppData\Local\Pub\Cache\hosted\pub.dev\google_mobile_ads-5.0.0\android\src\main\java\io\flutter\plugins\googlemobileads\AdMessageCodec.java:99: warning: [deprecation] MediationNetworkExtrasProvider in io.flutter.plugins.googlemobileads has been deprecated
      @Nullable MediationNetworkExtrasProvider mediationNetworkExtrasProvider) {
                ^
\AppData\Local\Pub\Cache\hosted\pub.dev\google_mobile_ads-5.0.0\android\src\main\java\io\flutter\plugins\googlemobileads\FlutterRequestAgentProvider.java:31: warning: [deprecation] getApplicationInfo(String,int) in PackageManager has been deprecated
              .getApplicationInfo(context.getPackageName(), PackageManager.GET_META_DATA);
              ^
\AppData\Local\Pub\Cache\hosted\pub.dev\google_mobile_ads-5.0.0\android\src\main\java\io\flutter\plugins\googlemobileads\GoogleMobileAdsPlugin.java:71: warning: [deprecation] MediationNetworkExtrasProvider in io.flutter.plugins.googlemobileads has been deprecated
  @Nullable private MediationNetworkExtrasProvider mediationNetworkExtrasProvider;
                    ^
10 warnings
```

**Comment by malandr2** [comment]
Hey @under3415, thanks for sharing. I've escalated this to the engineering team and expected to be fixed in the next release

**Comment by malandr2** [comment]
Closing as the PR was included in 5.1.0. Please open a new issue if deprecation warnings are visible!

**Comment by under3415** [comment]
Hi
You've missed one! Do you want me to open a new issue?

..\pub.dev\google_mobile_ads-5.1.0\android\src\main\java\io\flutter\plugins\googlemobileads\FlutterRequestAgentProvider.java:31: warning: [deprecation] getApplicationInfo(String,int) in PackageManager has been deprecated
              .getApplicationInfo(context.getPackageName(), PackageManager.GET_META_DATA);
              ^
